setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+
+setuptools.setup(
+    name="simGL", 
+    version="0.1",
+    author="Moisès Coll Macià, Graham Gower",
+    author_email="moicoll@birc.au.dk",
```

Comment:
Be aware that your email address will be farmed by spammers if you put it here. There's no requirement to provide an author_email, but it's up to you.